Source/WebKit/WebProcess/GPU/media/RemoteCDMFactory.cpp

```diff
@@ -39,8 +39,7 @@ namespace WebKit {
 
 using namespace WebCore;
 
-RemoteCDMFactory::RemoteCDMFactory(WebProcess& process)
-    : m_process(process)
+RemoteCDMFactory::RemoteCDMFactory(WebProcess&)
```

Comment:
Wait, why did you keep the parameter here? This looks like an oversight.

Reply:
OK, I see you've done this for classes that inherit from WebProcessSupplement, where it's needed. ack.